### PR TITLE
Moves contest listings from private to public wiki

### PR DESCRIPTION
**Pull request by tuupke**
The BAPC21, BAPC22, and NWERC22 contests have also been added. Though they might contain small mistakes.

### CODE REVIEW

**Review by vmcj — APPROVED** [review]
I think its fine to move this out of the private wiki, did not check if you added new text or altered any text.

File: content/wiki/contests/BAPC2019.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| Website    |              [link][website] |
+| Pictures   |               [link][photos] |
+| Scoreboard |           [link][scoreboard] |
+| Problemset |           [link][problemset] |
+| Teams      |                           56 |
+
+## Problemset
+| Contest     | Problemset                                                                         | Solutions                                                                         | Presentation                                                                      |
+|-------------|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| BAPC        | [link](http://commissies.ch.tudelft.nl/chipcie/archive/{year}/bapc/problemset.pdf) | [link](http://commissies.ch.tudelft.nl/chipcie/archive/{year}/bapc/solutions.zip) | [link](http://commissies.ch.tudelft.nl/chipcie/archive/{year}/bapc/solutions.pdf)
```
Comment (verwoerd) [inline content/wiki/contests/BAPC2019.md]: this seems to be going wrong. needs a replace `{year}` by `2019`

**Review by verwoerd — CHANGES_REQUESTED** [review]

File: content/wiki/contests/BAPC2019.md (hunk 1 above)
Comment (tuupke) [reply content/wiki/contests/BAPC2019.md]: Yes indeed, this was copied from the resources which was apparently already incorrect. This has now been fixed.

**Review by verwoerd — APPROVED** [review]

File: content/wiki/contests/NWERC2022.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+
+[home]: index.md
+[website]: https://2022.nwerc.eu/
+[email]: mailto:contact@2022.nwerc.eu
+[photos]: #
+[scoreboard]: https://2022.nwerc.eu/main/scoreboard/
+[problemset]: https://2022.nwerc.eu/test-session/problem-set.pdf
+[practiceset]: https://2019.nwerc.eu/files/nwerc2019practice.pdf
```
Comment (mpsijm) [inline content/wiki/contests/NWERC2022.md]:
```suggestion
```

File: content/wiki/contests/NWERC2022.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+| Solutions slides | [link](https://2022.nwerc.eu/main/solutions.pdf)                                  |
+| Problem packages | [link](https://2022.nwerc.eu/test-session/solutions.zip) |
+
+
+### Contest
+
+| Contest          | Problemset, solutions and presentation                   |
+|------------------|----------------------------------------------------------|
+| Problemset       | [link][problemset]                                       |
+| Solutions slides | [link](https://2022.nwerc.eu/test-session/solutions.pdf)  |
```
Comment (mpsijm) [inline content/wiki/contests/NWERC2022.md]:
```suggestion
| Problemset       | [link](https://2022.nwerc.eu/main/problem-set.pdf) |
| Solutions slides | [link](https://2022.nwerc.eu/main/solutions.pdf)   |
```

File: content/wiki/contests/NWERC2022.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Problemsets
+
+### Practice
+
+| Contest          | Problemset, solutions and presentation                                            |
+|------------------|-----------------------------------------------------------------------------------|
+| Problemset       | [link][practiceset]                                                               |
+| Solutions slides | [link](https://2022.nwerc.eu/main/solutions.pdf)                                  |
+| Problem packages | [link](https://2022.nwerc.eu/test-session/solutions.zip) |
```
Comment (mpsijm) [inline content/wiki/contests/NWERC2022.md]:
```suggestion
| Contest          | Problemset, solutions and presentation                     |
|------------------|------------------------------------------------------------|
| Problemset       | [link](https://2022.nwerc.eu/test-session/problem-set.pdf) |
| Solutions slides | [link](https://2022.nwerc.eu/test-session/solutions.pdf)   |
| Problem packages | [link](https://2022.nwerc.eu/test-session/solutions.zip)   |
```

**Review by mpsijm — CHANGES_REQUESTED** [review]
Found some typos in the URLs to the NWERC contest data :slightly_smiling_face:

File: content/wiki/contests/NWERC2022.md (hunk 4 above)
Comment (tuupke) [reply content/wiki/contests/NWERC2022.md]: These should be fixed by the latest commit.

**Review by mpsijm — APPROVED** [review]

**Pull request merged**